tests/integrations/controllers/WelcomeController.py

```diff
@@ -11,3 +12,43 @@ def test(self):
 
     def view(self, view: View):
         return view.render("welcome")
+
+    def create(self, view: View):
```

Comment:
@josephmancuso So here I am adding some controllers methods for the tests
and later in the test case I am adding those routes